doc/api/process.markdown

```diff
@@ -59,6 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
+This event may not be fired if the process terminates due to signals like
```

Comment:
I used the tentative `may` rather than `will` because I wasn't sure that what I was writing was true across all platforms.